Source/Server/Program.cs

```diff
@@ -22,14 +24,15 @@
 
 var configBuilder = new ConfigurationBuilder();
 configBuilder.AddJsonFile("appsettings.json");
-configBuilder.AddJsonFile("runtime.json");
-configBuilder.AddLegacyDolittleFiles();
+configBuilder.AddRuntimeFile();
+if (Directory.Exists(".dolittle"))
+{
+    configBuilder.AddLegacyDolittleFiles();
+}
```

Comment:
Two questions here:

1) Would it make sense to check for existence inside the legacy provider instead of here? That could also make it a little more robust if we check it for every file, instead of the whole directory.

2) As I understand how this code would work - if we update a current running microservice using the legacy files - the default new config in "runtime.json" would be loaded (with dev tenant and localhost mongodb), then all the actual config would be added. Meaning most microservices would end up with a new dev tenant configured on localhost mongodb. Which would probably fail the health checks once started up - and when projections/eventhandlers are registered, they would fail because it cannot connect to the EventStore. Right?

Reply:
1) yes we could do that
2) I agree, that sounds about right. One alternative could be to remove the dev tenant from the default config of the production image, and only have it in the dev image. This would be a breaking change. Then we could also start build an additional image which is a "dev-image without mongo", which essentialy would be the same as the current production images

Reply:
Hmm, would be nice if it is not breaking. Could we make up a non-breaking implementation somehow?

Reply:
I don't really see how we could since the configurations are added on top of each other and it makes sense for the legacy .dolittle files to override the config in runtime.json

Reply:
We could in the legacy file provider override clear everything under 'dolittle:runtime:tenants' if the tenants.json and/or resources.json files are present?